ui-admin/src/user/CreateUserModal.tsx

```diff
@@ -62,11 +79,17 @@ const CreateUserModal = ({ onDismiss, portals, userCreated }:
             </label>
           </div> }
           { !newUser.superuser && <div>
-            <label>
-              Portal
-              <Select options={portalOpts} value={selectedPortalOpt}
-                onChange={opt => setPortalShortcode(opt?.value ?? null)}/>
-            </label>
+            <div>
+              <label className="w-100 mb-3">
+                Portal
+                <Select options={portalOpts} value={selectedPortalOpt}
+                  onChange={opt => setPortalShortcode(opt?.value ?? null)}/>
+              </label>
```

Comment:
Not introduced in this PR, but I think we should hide this `Select` when you're adding a user from within the context of a portal

Reply:
I kept the select, but made it disabled (it only has one option in it anyway if you're within a portal context).  That way you still get the visual confirmation you're adding them to a specific portal, but it's clear you don't have a choice